proto/viam/service/slam/v1/slam.proto

```diff
@@ -123,6 +145,30 @@ message GetInternalStateRequest {
 }
 
 message GetInternalStateResponse {
-  // Internal state of the SLAM algorithm required to continue mapping/localization
+  // A chunk of the internal state of the SLAM algorithm required to continue
+  // mapping/localization
   bytes internal_state = 1;
 }
+
+message GetPointCloudMapStreamRequest {
```

Comment:
Not necessary to have anew request, can use the old one. Let us just be consistent with the final naming convention we wish to adopt for the Response (i.e. name contains stream or no)

Reply:
Unfortunately we need to create a new request & response type b/c `make lint` will fail the build if I try to reuse `GetPointCloudMapRequest`. Same goes for the other request / response message types.

Also re: https://github.com/viamrobotics/api/pull/196#discussion_r1108858538 (github won't let me comment on it for some reason), answer is yes.

Reply:
Got it. So during RSDK-1066 where we will be renaming and removing old functions, we will end up with:
- GetPointCloudMapRequest
- GetPointCloudMapResponse
- GetPointCloudMap
- GetInternalStateRequest
- GetInternalStateResponse
- GetInternalStateMap

Reply:
yes

Reply:
it's verbose but it's consistent!